--- origsrc/Wx-0.86/Wx.xs	2008-08-31 10:49:17.000000000 -0500
+++ src/Wx-0.86/Wx.xs	2008-09-24 01:56:31.578032800 -0500
@@ -276,10 +276,10 @@
         return;
 
     // not on first column to avoid tripping up xsubpp
-  #if defined(DEBUGGING) && !defined(PERL_USE_SAFE_PUTENV)
+#if defined(DEBUGGING) && !defined(PERL_USE_SAFE_PUTENV)
     // avoid crash on exit in Fedora (and other DEBUGGING Perls)
     PL_use_safe_putenv = 1;
-  #endif
+#endif
 
     int argc = 0;
 #if wxUSE_UNICODE && WXPERL_W_VERSION_GE( 2, 5, 3 )
--- origsrc/Wx-0.86/cpp/helpers.h	2008-08-31 10:49:17.000000000 -0500
+++ src/Wx-0.86/cpp/helpers.h	2008-09-24 01:46:06.578032800 -0500
@@ -444,7 +444,7 @@
 }
 
 #define WXPLI_BOOT_ONCE( name ) WXPLI_BOOT_ONCE_( name, XS )
-#if defined(WIN32) || defined(__CYGWIN__)
+#if defined(WIN32)
 #  define WXPLI_BOOT_ONCE_EXP( name ) WXPLI_BOOT_ONCE_( name, WXXS )
 #else
 #  define WXPLI_BOOT_ONCE_EXP WXPLI_BOOT_ONCE
