--- origsrc/Wx-0.84/Wx.xs	2008-06-29 14:16:27.000000000 -0500
+++ src/Wx-0.84/Wx.xs	2008-06-30 13:06:47.328125000 -0500
@@ -273,10 +273,10 @@
         return;
 
     // not on first column to avoid tripping up xsubpp
-  #ifdef DEBUGGING
+#ifdef DEBUGGING
     // avoid crash on exit in Fedora (and other DEBUGGING Perls)
     PL_use_safe_putenv = 1;
-  #endif
+#endif
 
     int argc = 0;
 #if wxUSE_UNICODE && WXPERL_W_VERSION_GE( 2, 5, 3 )
--- origsrc/Wx-0.84/cpp/helpers.h	2008-06-29 13:43:14.000000000 -0500
+++ src/Wx-0.84/cpp/helpers.h	2008-06-30 12:46:00.531250000 -0500
@@ -412,7 +412,7 @@
 }
 
 #define WXPLI_BOOT_ONCE( name ) WXPLI_BOOT_ONCE_( name, XS )
-#if defined(WIN32) || defined(__CYGWIN__)
+#if defined(WIN32)
 #  define WXPLI_BOOT_ONCE_EXP( name ) WXPLI_BOOT_ONCE_( name, WXXS )
 #else
 #  define WXPLI_BOOT_ONCE_EXP WXPLI_BOOT_ONCE
